### PR TITLE
cloud function support for handling inbound user SMS messages

### PR DESCRIPTION
**Pull request by un7c0rn**

### CODE REVIEW

File: backend/functions/sms/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Flask==1.0.2
```
Comment (dzx11) [inline backend/functions/sms/requirements.txt:1]: Wondering why we need Flask. Is it a requirement for twilio?

File: twilio/stv-twilio-service-test.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-    "account_sid": "xxxx",
-    "auth_token": "xxxx",
-    "notify_service_sid": "xxxx",
-    "phone_number": "xxxx"
-}
\ No newline at end of file
+    "account_sid": "ACcefc75688508570ea7dbe8da2eeeef40",
```
Comment (dzx11) [inline twilio/stv-twilio-service-test.json:2]: Are there any security risks from storing account details in plaintext? Wondering if there's a better way to store this

Comment (un7c0rn) [reply twilio/stv-twilio-service-test.json:2]: @dzx11 story keys on the server is ok, storing them in the codebase and on GitHub is not. haven't had time to look into the canonical solution for this in a public repo, so treating all keys as ephemeral/test for now. we have an issue tracking here https://github.com/un7c0rn/stopthevirus/issues/53.

File: backend/functions/sms/requirements.txt (hunk 1 above)
Comment (un7c0rn) [reply backend/functions/sms/requirements.txt:1]: @dzx11 removed in latest PR. thanks for catching!

**Pull request merged**